tests/com/haskforce/highlighting/annotation/external/GhcUtilTest.java

```diff
@@ -0,0 +1,53 @@
+package com.haskforce.highlighting.annotation.external;
+
+import com.intellij.openapi.editor.LogicalPosition;
+import com.intellij.openapi.editor.VisualPosition;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by kasper on 12/24/14.
+ */
+public class GhcUtilTest {
```

Comment:
Nice!  Thanks for adding tests.  Be sure to add this class to **tests/com/haskforce/HaskellTestCase.java**